uamqp/utils.py

```diff
@@ -93,7 +94,44 @@ def add_batch(batch, message):
     batch.data.append(output)
 
 
+def encode_str(data, encoding='utf-8'):
+    try:
+        return data.encode(encoding)
+    except AttributeError:
+        return data
+
+
+def normalized_data_body(data, **kwargs):
+    encoding = kwargs.get("encoding", "utf-8")
+    if isinstance(data, list):
```

Comment:
We shouldn't need these instance checks - where are these utilities called?

Reply:
it's called by the eventhub sdk to normalize data body